packages/css/README.md

```diff
@@ -1,27 +1,87 @@
-Apply the class names to your elements to make them stylable using design tokens.
+This package provides stylesheets for all components in the [Amsterdam Design System](https://designsystem.amsterdam) and some general utilities.
+Use it to apply the visual design of the City of Amsterdam to your HTML elements or non-React components.
 
-`npm install @amsterdam/design-system-css`
+## About
 
-Theoretically you can include the components in your HTML page like so, but this is not the typical use case of this library:
+The set of classes for a component can be seen as a blueprint for its markup and features.
+An `.ams-component` root selector applies the essential styles of the component to its element, and additional classes like `.ams-component--variant` modify its appearance or behaviour.
+
+Our [React components](https://www.npmjs.com/package/@amsterdam/design-system-react) use these classes in the HTML they render.
+You should use that package if your application uses React.
+
+The classes employ the [naming convention of NL Design System](https://nldesignsystem.nl/handboek/developer/architectuur#bem-class-names-voor-css).
+Other communities only need to overwrite design tokens to use our components with their branding.
+
+## Motivation
+
+Publishing the stylesheets as a separate package saves a lot of work when replicating components in other libraries, platforms, or Saas-applications.
+This is a powerful approach to implement the design system across different technologies.
+
+The classes act like a contract between the component’s definition and its implementations.
+As a bonus, updates to tokens or styles can be propagated to all of them.
+
+## Installation
+
+The stylesheets reference our [design tokens](https://www.npmjs.com/package/@amsterdam/design-system-tokens).
+Both packages need to be installed.
+
+```sh
+npm install
+  @amsterdam/design-system-css
+  @amsterdam/design-system-tokens
+```
+
+```sh
+pnpm add
+  @amsterdam/design-system-css
+  @amsterdam/design-system-tokens
+```
+
+```sh
+yarn add
+  @amsterdam/design-system-css
+  @amsterdam/design-system-tokens
+```
+
+## Usage
+
+### In JavaScript
+
+Import the main stylesheet and use the class names in your markup.
+
+```ts
+import "@amsterdam/design-system-tokens/dist/index.css";
+
+export const App = () => (
+  <div class="ams-screen">
+    <p className="ams-paragraph">Hello, world!</p>
+  </div>
+)
+```
+
+### In HTML
+
+Although it is not a typical use case, the stylesheet can be included in an HTML page directly.
 
 ```html
 <!doctype html>
 <html lang="en">
   <head>
-    <title>Example page</title>
-    <link rel="stylesheet" href="node_modules/@amsterdam/design-system-css" />
+    <title>Amsterdam Design System</title>
+    <link rel="stylesheet" href="node_modules/@amsterdam/design-system-css/dist/index.css" />
   </head>
-  <body>
+  <body class="ams-body">
     <p class="ams-paragraph">Hello, world!</p>
   </body>
 </html>
 ```
 
+### With CSS Modules
```

Comment:
Forgot to ask earlier: I lack experience with CSS modules, so I left the JS snippet below untouched. Can you check it @alimpens?

- Does it need the class component syntax?
- And the empty fragment?
- And the explicit return statement?
- How should we add the font and token imports?

Reply:
Hm, I missed this. I'm not sure we should keep this section at all. In Next and Vite the CSS file has to end in `*.module.css` to (automaticaly) be recognized as a CSS module, which means this example doesn't work with ADS and Vite / Next.

Reply:
Ah! Removed it.